velox/exec/AssignUniqueId.cpp

```diff
@@ -57,38 +60,43 @@ RowVectorPtr AssignUniqueId::getOutput() {
   if (input_ == nullptr) {
     return nullptr;
   }
-  generateIdColumn();
+  generateIdColumn(input_->size());
   auto output = fillOutput(input_->size(), nullptr);
   input_ = nullptr;
   return output;
 }
 
-void AssignUniqueId::generateIdColumn() {
-  auto numInput = input_->size();
+void AssignUniqueId::generateIdColumn(vector_size_t size) {
   auto result = results_[0];
   if (!result || !BaseVector::isReusableFlatVector(result)) {
-    result = BaseVector::create(BIGINT(), numInput, operatorCtx_->pool());
+    result = BaseVector::create(BIGINT(), size, operatorCtx_->pool());
```

Comment:
This is unrelated change, right?

Reply:
@mbasmanova this is unrelated. Just refactor

Reply:
It would be easier to review if a PR was limited to just a fix or just a refactor. Otherwise, it is hard to separate out material changes.

Reply:
@mbasmanova I will break this PR into 2